operator-common/src/main/java/io/strimzi/operator/common/AdminClientProvider.java

```diff
@@ -2,11 +2,11 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.operator.cluster.operator.resource;
+package io.strimzi.operator.common;
 
 import io.fabric8.kubernetes.api.model.Secret;
 import org.apache.kafka.clients.admin.AdminClient;
 
 public interface AdminClientProvider {
-    AdminClient createAdminClient(String hostname, Secret clusterCaCertSecret, Secret coKeySecret);
+    AdminClient createAdminClient(String hostname, Secret clusterCaCertSecret, Secret keyCertSecret, String keyCertName);
```

Comment:
KIP-476 added a superinterface called `Admin` for the `AdminClient` class, so I think we should make this interface return an `Admin`.